src/index.js

```diff
@@ -3,11 +3,13 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { BrowserRouter as Router } from 'react-router-dom'
+
 
 ReactDOM.render(
```

Comment:
Does this work? Because we normally use
const root = ReactDOM.createRoot(document.getElementById('root'));
root.render(
...
)

Reply:
Ignore this comment, I just saw that this is correct 😉